cmd/vic-machine/validate/validator.go

```diff
@@ -58,8 +58,18 @@ func (v *Validator) Validate(input *data.Data) (*metadata.VirtualContainerHostCo
 	vchConfig.ApplianceSize.Memory.Limit = int64(input.MemoryMB)
 	vchConfig.Name = input.DisplayName
 
+	var errs []string
 	if err := v.ParseComputeResourcePath(input.ComputeResourcePath); err != nil {
-		return nil, err
+		errs = append(errs, err.Error())
```

Comment:
Do not call getConnection here. Instead, you can bypass the later parameter settings from derived datacenter name and cluster name.
Actually, those code should be refactored to set only when the relied parameter is not "".
Then it's possible to suggest value for them in the future.